ddtrace/internal/utils/deprecations.py

```diff
@@ -0,0 +1,8 @@
+class DDTraceDeprecationWarning(DeprecationWarning):
+    pass
+
+
+# Override module to simplify adding warning filters by querying for
+# ddtrace.DDTraceDeprecationWarning but not have to expose this in the
+# public API.
+DDTraceDeprecationWarning.__module__ = "ddtrace"
```

Comment:
I think we could just declare this in `ddtrace.__init__` since we import it there and expose it anyway

Reply:
That approach results in circular imports. We use ``DDTraceDeprecationWarning` in ``ddtrace.tracer`` and ``ddtrace.config``

Reply:
Ah probably worth documenting that as well here then